Person.rb

```diff
@@ -0,0 +1,48 @@
+class Person
+  att_accessor :id, :name, :age
+  def initialize(name='Unknown', age, parent_permission=false)
+    self.age = age
+    self.name = name
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] kindly add id to your initialize method  i.e ` @id = Random.rand(1..1000)`

Reply:
Thank you for the feedback. I will implement the suggestions.